generate_tools.go

```diff
@@ -11,8 +11,13 @@ package tools
 // This is the currently recommended approach: https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 
 import (
+	_ "github.com/go-delve/delve/pkg/goversion"
```

Comment:
I'm not sure if these dependencies should go here, and if I should modify the previous PR (#10039) to include its dependencies as well